pkg/rbac/clusterrole.go

```diff
@@ -78,30 +78,6 @@ func (builder *ClusterRoleBuilder) WithRules(rules []v1.PolicyRule) *ClusterRole
 		return builder
 	}
 
-	for _, rule := range rules {
-		if len(rule.APIGroups) == 0 {
-			glog.V(100).Infof("The clusterrole rule must contain at least one APIGroup entry")
-
-			builder.errorMsg = "clusterrole rule must contain at least one APIGroup entry"
-		}
-
-		if len(rule.Verbs) == 0 {
```

Comment:
Seems that Verbs is a required field. I think we should keep it

Reply:
yes, I think you're right, verbs verification should be left